src/pages/registration/contributors_tab/components/ContributorRow.tsx

```diff
@@ -176,11 +183,11 @@ export const ContributorRow = ({
           {!contributor.identity.id && (
             <Button
               variant="outlined"
-              sx={{ textTransform: 'none' }}
+              sx={{ textTransform: 'uppercase', paddingY: '0.1rem', paddingX: '0.75rem' }}
```

Comment:
`uppercase` er default på buttons (via theme).
Syns også alignment så riktig ut bare ved å sette size med meg. Ellers kan det ev kortes ned til `p: '0.1rem 0.75rem'` også
```suggestion
              size="small"
```

Reply:
Keeping the padding instead of small because I feel the alignment gets so ugly it burns my eyes with "small". But willing to discuss it in a meeting if you disagree. Fixed the shorthand form and removed uppercase.